### ISSUE TITLE
Does the plugin print a string of text?

### ISSUE THREAD

**Issue by felipeblan** [envisiondata/cordova-plugin-starprinter#2]
The plugin only prints a predesigned receipt?

**Comment by envisiondata** [comment]
That is correct. This is just to get you started. The receipt that prints is the 3" sample receipt that comes with the SDK from Star Micronics. You will need to customize for your own application. To do this you will need to update the methods in the src folder passing in you receipt data.
You can find documentation for the Star SDK here. Star Global Support Site

|
|

Star Global Support Site
   |  |

  |

Hope this gets you started in the right direction.

```
  From: Felipe <notifications@github.com>
```

 To: envisiondata/cordova-plugin-starprinter cordova-plugin-starprinter@noreply.github.com 
 Sent: Monday, May 30, 2016 9:25 AM
 Subject: [envisiondata/cordova-plugin-starprinter] Does the plugin print a string of text? (#2)

The plugin only prints a predesigned receipt?—

**Comment by elie-chaaban** [comment]
how can i get this plugin and change in java file and use it? im trying but it s not working

**Comment by elie-chaaban** [comment]
it s always printing the sample receipt even thou i changed it in PrintSampleReceipt
what can i do

**Comment by envisiondata** [comment]
That is all it does, except print a signature. This plugin is just to get you started so you don't have to build one from scratch.
Check out this plugin. I think, but am not sure it is just like mine.
InteractiveObject/StarIOPlugin

  
|  
|   
|   
|   |    |

   |

  |
|  
|    |  
InteractiveObject/StarIOPlugin
 StarIOPlugin - Star Micronics printer phonegap/cordova plugin  |   |

  |

  |

**Comment by envisiondata** [comment]
Also here is a search that might help.
https://github.com/search?utf8=%E2%9C%93&q=Star+Micronics&type=

**Comment by envisiondata** [comment]
If your using Cordova just use the CLI like the instructions say. What kind of error are you getting? Can you install any plugin?

You will have to do some custom development with this plugin. You will have to pass your data and change the iOS and android native code to read that data.

James

**Comment by elie-chaaban** [comment]
no, eveything worked but one thing more
i want to pass paramters to PrintSampleReport in the [ID] parameter
an array or anything
how can i read them in Java in PrintSampleReport function? can you help me please

**Comment by elie-chaaban** [comment]
lets say i want to put the header of the receipt from parameter
can you help me how i can pass them?

**Comment by elie-chaaban** [comment]
I did it
thanks you rock!
